cmd/admin-handlers.go

```diff
@@ -244,39 +245,111 @@ func (a adminAPIHandlers) ServerInfoHandler(w http.ResponseWriter, r *http.Reque
 	if objectAPI == nil {
 		return
 	}
+	var jsonBytes []byte
+	var err error
+
+	vars := mux.Vars(r)
+	switch infoType := vars["infoType"]; infoType {
+
+	case "vault":
+		if GlobalKMS == nil {
+			writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErr(ErrKMSNotConfigured), r.URL)
+			return
+		}
+		config, err := readServerConfig(ctx, objectAPI)
+		if err != nil {
+			writeErrorResponseJSON(ctx, w, toAdminAPIErr(ctx, err), r.URL)
+			return
+		}
+		var Environment = environment{}
```

Comment:
What is the variable `environment` used for?

Reply:
removed.